x-pack/plugins/apm/server/routes/typings.ts

```diff
@@ -64,9 +65,16 @@ export interface Route<
   handler: RouteHandler<TRouteParamsRT, TReturn>;
 }
 
+/**
+ * @internal
+ */
+export interface AplPluginRequestHandlerContext extends RequestHandlerContext {
```

Comment:
I think this is supposed to be `ApmPluginRequestHandlerContext` not `Apl`.

Reply:
renamed